provisions/roles/jenkins/slave/tasks/cleanup_registry.yml

```diff
@@ -3,5 +3,5 @@
 - name: Copy cron script to clean up mismatched tags from registry
   template: src=cron_delete_index_mismatch.sh.j2 dest=/root/cron_delete_index_mismatch.sh mode=0755
 
-#- name: Add cronjob to clean up mismatched tags from registry
-#  cron: name="To delete mismatched containers from registry" hour="*/24" job=/root/cron_delete_index_mismatch.sh
\ No newline at end of file
+- name: Add cronjob to clean up mismatched tags from registry
+  cron: name="To delete mismatched containers from registry" hour="*/24" job=/root/cron_delete_index_mismatch.sh
```

Comment:
Make sure the contents and parameters of cron template matches to latest changes in gc

For now, it reads like:
```
/usr/bin/python /opt/cccp-service/container_pipeline/cleanup_registry/__init__.py -cr {{ public_registry }} -p 5000 \
    -i {{ cccp_index_repo }};
```

Reply:
The core parameters were never changed so we should be good in that regard.